lancero/lancero.go

```diff
@@ -193,6 +193,7 @@ func OdDashTX(b []byte, maxLines int) string {
 	outBuffer.WriteString("L = least significant byte, M = most significant byte\n")
 	outBuffer.WriteString("framebit in fbkL\n")
 	outBuffer.WriteString("errLerrM fbkLfdbM errLerrM fbkLfdbM errLerrM fbkLfdbM errLerrM fbkLfdbM \n")
```

Comment:
Are these right once the bits are flipped to make words read little endian?

Reply:
I'll check. I think they should read "fMfLeMeL"

I'll fix.